content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
```

Comment:
```suggestion

{{< feature-state for_k8s_version="v1.19" state="alpha" >}}

```
(this is where you could link to the text about enabling CSIStorageCapacity)

Reply:
I'm not sure I understand the comment - should I link to the "Enabling the feature" section here?

Reply:
That's what I'd do @pohly - use a link inside a note shortcode to clearly highlight that it needs enabling, but let a reader's eyes naturally move on to learning about what the thing is that they might want to enable.

Reply:
The key thing is that there's a convention for marking features as alpha / beta and this page should follow that convention.

(can fix that in a follow-up PR though).

Reply:
You convinced me that the "enabling" section is better off at the end of the page :grin: I've moved it.

Regarding conventions for alpha features, does the page now follow those?